sympy/physics/vector/functions.py

```diff
@@ -587,20 +587,33 @@ def dynamicsymbols(names, level=0):
     level : int
         Level of differentiation of the returned function; d/dt once of t,
         twice of t, etc.
+    Assumptions :
+        real = False
+        positive = False
```

Comment:
This should say something like "Key-value pairs of assumptions, see `symbols` for more information". You should read this: https://docs.sympy.org/dev/documentation-style-guide.html

Reply:
Actually I have referred it from class Symbol, and it had similar kind to explanation of assumption

Reply:
Well, that sounds like Symbol's docstring is poor then and we should make it better here.